single.just

```diff
@@ -13,7 +13,7 @@ simulate hdPath='0':
   #!/usr/bin/env bash
   bundlePath=${taskPath}/${bundleName}.json
   echo "getting signer address..."
-  if [ t"$SIMULATE_WITHOUT_LEDGER" == t1 ]
+  if [ ! -z "$SIMULATE_WITHOUT_LEDGER" ]
```

Comment:
The previous command doesn't work, or this is an improvement because it'll accept any non-empty value? 

Can you also apply this in nested.just?

Reply:
I am unaware of the syntax that was used previously. What did you intent for it to do? Now it simply says "if the variable is not zero" which seems like better UX in this case

Reply:
Added a commit doing the same on `nested.just`